### DISCUSSION TITLE
Get 400 No required SSL certificate was sent when installing signserver

### DISCUSSION THREAD

**Discussion by quyen66** [Keyfactor/ejbca-ce#432] [Q&A]
I'm following this tutoril https://www.youtube.com/watch?v=nntAbPvxQAg to install signserver, but when access the url it return me 400 No required SSL certificate. I knew the problem cause by the cert but not sure what step I did wrong?

![Screenshot 2023-12-05 142802](https://github.com/Keyfactor/ejbca-ce/assets/74227977/a4b4ad8f-448e-4ae5-ad6f-14c10ef75478)

![Screenshot 2023-12-05 142824](https://github.com/Keyfactor/ejbca-ce/assets/74227977/ebfd271c-e4ef-454c-855c-f64a4d283230)

![Screenshot 2023-12-04 103851](https://github.com/Keyfactor/ejbca-ce/assets/74227977/e3be5561-be5b-4fbd-8752-25006db2490c)

**Comment by primetomas** [comment]
You can start by following this tutorial here: https://doc.primekey.com/signserver/tutorials-and-guides/quick-start-guide-start-signserver-container-with-client-certificate-authenticated-access

It's a bit more basic with all the steps. 
First thing you must have done is to issue a client authentication certificate and install it in your browser. I.e. the link to https://doc.primekey.com/ejbca/tutorials-and-guides/quick-start-guide-issue-client-authentication-certificate-using-ejbca from the tutorial. You will use this client certificate to access SignServer. 

I hope that helps.

**Reply by quyen66** [reply]
I'm working with helm and k8s so I use helm for build up signserver, but thanks btw. I just can't understand clearly about ssl. Following the youtube, he use tls server to issue the signserver right? I also created tls server but why it had error? Or my certificated created wrong somewhere?

**Reply by primetomas** [reply]
Perhaps it would be better to hold this question in the SignServer forum. https://github.com/Keyfactor/signserver-ce/discussions
You also need a client certificate, but it depends on the signserver startup, and I'm not familiar with that helm process.

**Reply by quyen66** [reply]
Sure, thanks.
